test/integration/nighthawk_grpc_service.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import os
+import socket
+import subprocess
+import sys
+import tempfile
+import threading
+import time
+from string import Template
+
+from common import IpVersion
+
+
+class NighthawkGrpcService(object):
+  """
+    Base class for running a server in a separate process.
+    """
+
+  def __init__(self, server_binary_path, server_ip, ip_version):
```

Comment:
Can we add a comment for the initializer documenting its arguments? Something like:

def __init__(self, server_binary_path, server_ip, ip_version):
  """Initializes Nighthawk gRPC service.

  Args:
    server_binary_path: A string, path to the server binary.
    ...
  """